docs/developer-tutorials/local-test-setup.md

```diff
@@ -88,8 +91,14 @@ This setup is deployed with the following security features enabled:
    Adjust the path to your `template-operator` local directory or any other reference module operator accordingly.
 
    ```shell
-   kyma alpha create module -p ../template-operator --version 1.2.3 \
-   --registry k3d-registry.localhost:5111 --insecure --module-config-file ../template-operator/module-config.yaml
+   cd <local path to template operator repository>
+   
+   # generates the manifests and saves them to a template-operator.yaml file
+   make build-manifests
+   
+   # create the a ModuleTemplate CR and saves it to a template-yaml file
+   kyma alpha create module -p ./ --version 1.2.3 \
+   --registry k3d-registry.localhost:5111 --insecure --module-config-file ./module-config.yaml
```

Comment:
comment: at least on my machine I cannot just create the module specifying a path to the template operator repo. Therefore I changed this to cd into the needed directory.

E.g. (`../template-operator` is my local way to the repo): 

```sh
kyma alpha create module -p ../template-operator --version 1.2.3 \
   --registry k3d-registry.localhost:5111 --insecure --module-config-file ../template-operator/module-config.yaml
CAUTION: You're using an outdated version of the Kyma CLI (2.20.1). The latest released version is: 2.20.3

WARNING: This command is experimental and might change in its final version. Use at your own risk.
- Module built (8ms)
- Default CR validation succeeded (9.595s)
- component descriptor created (1ms)
X Configuring security scanning... (5ms)
Error: component.resources.0: Must validate at least one schema (anyOf);component.resources.0.version: Does not match pattern '^[v]?(0|[1-9]\d*)(?:\.(0|[1-9]\d*))?(?:\.(0|[1-9]\d*))?(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$'
```